setup.py

```diff
@@ -57,7 +57,7 @@ def get_version(file="flake8_isort.py"):
     zip_safe=False,
     install_requires=[
         'flake8 >= 3.2.1, <4',
-        'isort[pyproject] >= 4.3.5, <5',
+        'isort[pyproject] >= 4.3.5, <5',  # https://github.com/gforcada/flake8-isort/issues/88
```

Comment:
Actually, that makes the line too long. flake8 will complain.
```suggestion
        'isort[pyproject] >= 4.3.5, <5',
```